bc_obps/reporting/schema/report_new_entrant.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
```

Comment:
In the schema, we could use a ninja resolver (docs here https://django-ninja.dev/guides/response/) to take a list of annotated EmissionCategory records and group them by emission category type